apolloschurchapp/app.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
I think we still need this file.

Reply:
https://github.com/react-native-community/cli/issues/1113

Reply:
don't think we do. it came from expo, was just to pull the name from inside the index.js file (which ours doesn't do)

Reply:
I ran rn init recently, let me dig up what it included....

Reply:
okay - so the default react-native app does create that app.json.

Reply:
ah. so according to that issue, it's created on init but not used anymore. I can put it back, it's just another file churches *should* rename even though it seems unnecessary

Reply:
was trying to avoid as much renaming as possible

Reply:
that's fair - none of our churches have ever renamed it to my knowledge so I'm fine with deleting.